exonum-java-binding-core/pom.xml

```diff
@@ -144,7 +144,10 @@
           <excludes>
             <exclude>**/*IntegrationTest.java</exclude>
           </excludes>
-          <argLine>${java.vm.assertionFlag}</argLine>
+          <argLine>
+            ${argLine}
```

Comment:
I don't see where the 'argLine' property is set. Is it a standard one?

Reply:
jacoco:prepare-agent prepares it.
> Prepares a property pointing to the JaCoCo runtime agent that can be passed as a VM argument to the application under test. Depending on the project packaging type by default a property with the following name is set:
tycho.testArgLine for packaging type eclipse-test-plugin and
argLine otherwise.

https://www.eclemma.org/jacoco/trunk/doc/prepare-agent-mojo.html